xline-test-utils/src/lib.rs

```diff
@@ -54,12 +54,12 @@ impl Cluster {
     }
 
     #[allow(dead_code)] // used in tests but get warning
```

Comment:
This problom seems to no longer exist, we can remove this line.

Reply:
done